OpenRA.Mods.Common/Traits/World/ActorMap.cs

```diff
@@ -261,7 +261,8 @@ public IEnumerable<Actor> GetActorsAt(CPos a)
 			if (!influence.Contains(uv))
 				return Enumerable.Empty<Actor>();
 
-			var layer = a.Layer == 0 ? influence : customInfluence[a.Layer];
+			var layerIndex = a.Layer;
+			var layer = layerIndex == 0 ? influence : customInfluence[layerIndex];
```

Comment:
Fun fact: It did look like the jit compiler was able to do this optimization by itself.

Reply:
Should we drop these then?